hoomd/md/methods/rattle.py

```diff
@@ -138,6 +138,67 @@ def _attach(self):
         super()._attach()
 
 
+class DisplacementCapped(NVE):
+    r"""NVE-like integration with capped displacement.
+
+    Integration is via a maximum displacement capped Velocity-Verlet with
+    RATTLE constraint. This class is useful to relax a simulation on a manifold.
+
+    Args:
+        filter (hoomd.filter.filter_like): Subset of particles on which to apply
+            this method.
+        maximum_displacement (hoomd.variant.variant_like): The maximum
+            displacement allowed for a particular timestep.
+        manifold_constraint (hoomd.md.manifold.Manifold): Manifold
+            constraint.
+        tolerance (`float`, optional): Defines the tolerated error particles are
+            allowed to deviate from the manifold in terms of the implicit
+            function. The units of tolerance match that of the selected
+            manifold's implicit function. Defaults to 1e-6
+
+    `DisplacementCapped` performs constant volume simulations as described in
+    `hoomd.md.methods.DisplacementCapped`. In addition the particles are
+    constrained to a manifold by using the RATTLE algorithm.
+
```

Comment:
Add a warning block stating that displacement capped dynamics are not energy or momentum conserving.